RadarSDKTests/Resources/geocode_ip.json

```diff
@@ -1,10 +1,28 @@
 {
```

Comment:
is this payload not keyed with the `address` field?

Reply:
another thought - do we need this `.json` file anymore? If the response is consistent with the other geocoding endpoints, we might be able to use `geocode.json` as a fixture everywhere

Reply:
I think we do since IP returns a single address and forward/reverse return an array